the-open-university-harvard-s390-modules.csl

```diff
@@ -5,7 +5,7 @@
     <id>http://www.zotero.org/styles/the-open-university-harvard-s390-modules</id>
     <link href="http://www.zotero.org/styles/the-open-university-harvard-s390-modules" rel="self"/>
     <link href="http://www.zotero.org/styles/the-open-university-harvard" rel="template"/>
-    <link href="https://learn1.open.ac.uk/mod/oucontent/view.php?id=2620" rel="documentation"/>
+    <link href="http://www.open.ac.uk/libraryservices/documents/Harvard_citation_hlp.pdf" rel="documentation"/>
```

Comment:
@neallyon, just to make sure, it looks like http://www.open.ac.uk/libraryservices/documents/Harvard_citation_hlp.pdf describes the general Open University Harvard style, right? Does https://learn1.open.ac.uk/mod/oucontent/view.php?id=2620 link to the document that describes the specific requirements of S390?

Reply:
@rmzelle, 
(1) Yes, the linked pdf is a publicly available reference to the general style.
(2) No, the second link is to the same general style, only in a HTML format on a site requiring login. 

The specific referencing requirements for S390 (which is an 'umbrella term' for a small suite of science project modules of different disciplines achieving essentially the same objectives so managed under the same module rather than through the specific science faculties) are provided within module materials which are only available on a site requiring login, as far as I am aware. Unfortunately as the Open University is a pure distance-learning provider, a lot of their material is HTML-based and only available via their webpages which require login.  

These specific requirements are very minor; my adjustments simply ensure papers with more than 10 authors show one author and et al (in italics) in the citations and show only the first 10 authors and et al (in italics) in the bibliography - the original shows all authors in the bibliography and et-al is not in italics. Having adjusted the file myself last year, I have passed it to students this year, but thought it would be a good idea to make it an official .csl in the Zotero repository for future students.

Reply:
@rmzelle  -- this sounds OK by me: will merge in ~48 hs unless you voice concerns, with a link to the above comment in the documentation.